File: platforms/python/examples/scene/model/translate.py
from fragmentcolor import Material, Mesh, Model, Renderer, Vertex

renderer = Renderer()
mesh = Mesh()
mesh.add_vertex(
    Vertex([0.0, 0.0, 0.0]).set(Vertex.NORMAL, [0.0, 1.0, 0.0]).set(Vertex.UV0, [0.0, 0.0]),
)

model = Model(mesh, Material.pbr(renderer))
model.translate([5.0, 0.0, -2.0])

File: platforms/web/examples/scene/model/translate.js
import { Material, Mesh, Model, Renderer, Vertex } from "fragmentcolor";

const renderer = new Renderer();
const mesh = new Mesh();
mesh.addVertex( Vertex.new([0.0, 0.0, 0.0]) .set(Vertex.NORMAL, [0.0, 1.0, 0.0]) .set(Vertex.UV0, [0.0, 0.0]), );

const model = new Model(mesh, Material.pbr()?);
model.translate([5.0, 0.0, -2.0]);
